src/index.html

```diff
@@ -9,10 +22,51 @@
     <title>Product cards</title>
     <link
       rel="stylesheet"
-      href="./styles/index.scss"
+      href="./styles/styles.scss"
     />
   </head>
   <body>
-    <h1>Product cards</h1>
+    <div
+      class="card"
+      data-qa="card"
+    >
+      <header class="card__header">
+        <img
+          class="card__header-logo"
+          alt="logo"
+          src="./images/imac.jpeg"
+        />
+      </header>
+      <h1 class="card__title-1">
```

Comment:
This tag used for title of the full page, so use the another tag here
```suggestion
      <h3 class="card__title">
```